.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+For this PR, I have:
+<!-- aports checklist -->
+<!-- You can remove any items that don't apply. This is a helpful starting point. -->
+- [ ] Prefixed repo/name: directory in my commit title line
+- [ ] Started `pkgrel` at `0` for a new version or increased `pkgrel`
+- [ ] Used "upgraded to" or "new aport" phrase in title line
+- [ ] Used global `depends_`_split_ variables (e.g. `depends_dev` or `$pkgname-foo:_foo` => `depends__foo`)
```

Comment:
How did you came to this? abuild supports only `depends_dev`, that's a special case.

Reply:
yes, this line needs to be removed as it is wrong.

Reply:
I put that line in based on previous advice from @ncopa  that depends need to be defined globally rather than in split functions.

I understand that `default_dev` has a depends line using `depends_dev` already. Extending this pattern just makes the most sense to me for custom split functions too.